### ISSUE TITLE
alpha should work on line geom_sf()

### ISSUE THREAD

**Issue by yutannihilation** [tidyverse/ggplot2#3589]
``` r
library(ggplot2)
library(sf)
#> Linking to GEOS 3.7.2, GDAL 3.0.1, PROJ 6.2.0

d <- data.frame(x = 1:5, y = 1:5)

l <- st_linestring(as.matrix(d))
sf <- st_sf(geometry = st_sfc(l))

p1 <- ggplot(d, aes(x, y)) + geom_line(alpha = 0.1) + ggtitle("geom_line()")
p2 <- ggplot(sf) + geom_sf(alpha = 0.1) + ggtitle("geom_sf()")

patchwork::wrap_plots(p1, p2)
```

![](https://i.imgur.com/FQmzJqr.png)

<sup>Created on 2019-10-27 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by yutannihilation** [comment]
I'll fix this when #3546 gets merged. It would be easy as just adding `| is_line` here:

https://github.com/tidyverse/ggplot2/blob/6f5ffea70e80ce3aa076fb9da8a6b0ec7ebfbc51/R/geom-sf.R#L152

**Comment by thomasp85** [comment]
yeah - that is an oversight on my part. Good catch, and your fix should work nicely
